### ISSUE TITLE
Using BRIO for text summarization on another language

### ISSUE THREAD

**Issue by thaokimctu** [yixinL7/BRIO#24]
Good day Mr. Yixin, thank you so much for your work.
I'm currently trying to use pretrained MBart model as baseline and custom dataset in order to implement BRIO on Vietnamese. I replaced crucial things like using mbart_modeling.py, changing BartScorer to MBartScorer, use suitable tokenizer,... but the ROUGE scores during training was quite poor:
![image](https://user-images.githubusercontent.com/111738694/192474938-3b1cc2ad-266e-46d8-809e-d6e847eeedcb.png)
I tried to config three most sensitive parameters like you suggested in this post https://github.com/yixinL7/BRIO/issues/9 but to no avail. The MLE avg loss is always around 6.0 at the beginning of training:
![image](https://user-images.githubusercontent.com/111738694/192491352-013637c5-0099-49f2-89c8-3cfbf6f3423b.png)
Can you give me some advice?

**Comment by MinhDang685** [comment]
Hi @thaokimctu, did you find a solution for your problem? I'm in the same situation as you was, I try to re-implement BRIO for 
a Vietnamese dataset (Wikilingua) with a T5 model. 

But after the training (got avg ranking loss: 0.019404, avg mle loss: 3.885741 after 8 epochs), the ROUGE scores has been dropped dramatically, and the model generated outputs have lots of duplication and unrelated words.

**Comment by thaokimctu** [comment]
@MinhDang685, Let's discuss in Vietnamese :)
Ở trường hợp của mình thì mình có tham khảo ý kiến của tác giả mô hình Bartpho và quyết định finetune mô hình qua tập dữ liệu một lần (bằng cách sử dụng script finetune cho text summarization của huggingface) rồi mới áp dụng mô hình huấn luỵện BRIO. 
Ngoài ra, bước tiền xử lý dữ liệu cũng quan trọng, mình gặp trục trặc ở bước tiền xử lý dữ liệu là nhỡ set số lượng candsum sinh ra cho 3 tập test,val,eval không bằng nhau, bạn có thể kiểm tra ở bước này.
Chúc bạn thành công.

**Comment by MinhDang685** [comment]
@thaokimctu Hi Thảo, cám ơn bạn nhiều nha.
> finetune mô hình qua tập dữ liệu một lần

mình bị thiếu bước này, đang thử fine tune rồi chạy generate candidate lại. 

Nếu được bạn cho mình xin địa chỉ mail mình hỏi thêm chút được không.

**Comment by thaokimctu** [comment]
@MinhDang685 Vì vài lí do nên mình không hướng dẫn bạn tiếp được. Cố lên =))
